packages/evolution-generator/src/scripts/conditionals_generator.py

```
@@ -226,6 +270,24 @@ def _collect_row_validation_issues(
+            and raw_path is not None
+            and isinstance(raw_path, str)
+        ):
+            tokens = ConditionalsGenerator._path_expansion_tokens_found(
+                raw_path,
+                current_context_specs=ConditionalsGenerator.CONDITIONALS_CURRENT_CONTEXT_SPECS,
+            )
+            if len(tokens) > 1:
+                issues.append(
+                    f"{prefix}Invalid path in row {row_number}: "
+                    "only one expansion token is allowed in the path; "
+                    f"found {', '.join(tokens)} in {repr(raw_path)}"
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)